### DISCUSSION TITLE
What stays...and what goes

### DISCUSSION THREAD

**Discussion by johnmackintosh** [johnmackintosh/rockthemes#15] [Q&A]
I want to keep this package manageable, so while it's tempting to just keep adding to it, I am also thinking about 1in, 1 out.
To that end...

1. What is your favourite?

2. What is your least favourite?

To make it simpler, I will assume that  if you don't like the 4 tone palette , you don't like the associated 10 tone palette either. So just list the corresponding 4 tone palette for now.

Thanks 👍

**Comment by softloud** [comment]
deee_lite goes! :croissant: meloncollie stays!
